src/extension/netcdf-out/src/test/java/org/geoserver/wcs2_0/MultiDimDataStoreFactory.java

```diff
@@ -35,4 +49,19 @@ public String getDescription() {
         return new DataStoreFactorySpi.Param[] { new Param("ParentLocation") };
     }
 
+    @Override
+    public boolean canProcess(Map<String, Serializable> map) {
+        File dir = new File((String) map.get("ParentLocation"));
+        return dir.exists();
```

Comment:
There is a utility method to change the map against parameter info you could use.